.ci-config/rippled.cfg

```diff
@@ -35,26 +23,13 @@ protocol = ws
 [node_size]
 small
 
-# tiny
-# small
-# medium
-# large
-# huge
-
 [node_db]
-type=NuDB
-path=/var/lib/rippled/db/nudb
-advisory_delete=0
-
-# How many ledgers do we want to keep (history)?
```

Comment:
Why are all the comments removed?

Reply:
I've explained in point-2 under "Context of Change": https://github.com/XRPLF/xrpl-py/pull/824#issue-2978113875

Reply:
The comments are for us, not for rippled - they're _supposed_ to be ignored by rippled